package.json

```diff
@@ -25,6 +25,7 @@
     "ember-cli-htmlbars": "^1.1.1",
     "ember-cli-htmlbars-inline-precompile": "^0.3.6",
     "ember-cli-inject-live-reload": "^1.4.1",
+    "ember-cli-moment-shim": "^3.1.0",
```

Comment:
You need to run a `yarn install` to get these new dependencies added to the `yarn.lock` file.